### PR TITLE
feat(core/users) Material Design

### PR DESCRIPTION
**Pull request by siteshackinc**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -6,13 +6,17 @@
   "dependencies": {
     "angular": "~1.5",
     "angular-animate": "~1.5",
+    "angular-file-upload": "1.1.5",
```
Comment (trainerbill) [inline bower.json:9]: Do we need this right now?  Do you have the profile image upload working?  I would rather switch to ng-file-upload as it has more of a following

File: modules/core/client/app/core.client.app.loader.js [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
   window.modernMeanApplication = {
     name: 'modernMean',
-    dependencies: ['ngResource', 'ui.router', 'ngSanitize', 'ngMaterial'],
+    dependencies: ['ngResource', 'ui.router', 'ngSanitize', 'ngMaterial', 'ngMessages', 'ngMdIcons', 'angularFileUpload'],
```
Comment (trainerbill) [inline modules/core/client/app/core.client.app.loader.js:6]: remove angularFileUpload

File: modules/users/client/views/settings/manage-social-accounts.client.view.html [hunk 3]
```diff
@@ -1,47 +1,47 @@
-<section class="row">
-  <h3 class="col-md-12 text-center" ng-show="vm.hasConnectedAdditionalSocialAccounts()">Connected social accounts:</h3>
+<md-content>
+  <h1 class="md-display" ng-show="vm.hasConnectedAdditionalSocialAccounts()">Connected social accounts:</h1>
   <div class="col-md-12 text-center">
     <div ng-repeat="(providerName, providerData) in vm.user.additionalProvidersData" class="social-account-container">
-      <img ng-src="/dist/img/users/client/img/buttons/{{providerName}}.png">
+      <img ng-src="/modules/users/client/img/buttons/{{providerName}}.png">
```
Comment (trainerbill) [inline modules/users/client/views/settings/manage-social-accounts.client.view.html:5]: I think these need to be /dist/img.  /modules is not longer accessible by the web.  Only the public folder

**Comment by trainerbill** [comment]
Fix what I commented on and merge.  Nice work

File: bower.json (hunk 1 above)
Comment (siteshackinc) [inline bower.json:9]: Yes it works!, I will look at using ng-file-upload

**Pull request merged**